definitions/npm/@testing-library/react_v9.x.x/flow_v0.104.x-/react_v9.x.x.js

```diff
@@ -115,182 +115,204 @@ declare module '@testing-library/react' {
     eventProperties?: TInit
   ) => boolean;
 
-  declare type RenderResult = {|
+  declare type RenderResult<Queries = GetsAndQueries> = {|
     container: HTMLDivElement,
     unmount: () => void,
     baseElement: HTMLElement,
     asFragment: () => DocumentFragment,
     debug: (baseElement?: HTMLElement) => void,
     rerender: (ui: React$Element<*>) => void,
-  |} & GetsAndQueries;
+  |} & Queries;
```

Comment:
So we use an union on an exact object, and spread on an inexact one... 
If all works as intended okay, but feels like we really need to rewrite the whole definition.